src/main/java/Duke.java

```diff
@@ -1,10 +1,58 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static String EXIT_CMD = "bye";
+    public static String LIST_CMD = "list";
+    public static String DONE_CMD = "done";
+    public static String TODO_CMD = "todo";
+    public static String DEADLINE_CMD = "deadline";
+    public static String EVENT_CMD = "event";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        greetUserOnStart();
+        Scanner input = new Scanner(System.in);
+
+        while (true) {
+            String command = input.nextLine();
+
+            // parse and handle commands
+            if (command.equals(EXIT_CMD)) {
+                break;
+            } else if (command.equals(LIST_CMD)) {
+                TaskManager.listTasks();
+            } else if (command.startsWith(DONE_CMD)) {
+                String parsedInput = command.split(" ")[1];
+                int taskNo = Integer.parseInt(parsedInput);
+                TaskManager.markTaskNoAsDone(taskNo - 1);
```

Comment:
Perhaps you might consider extracting out the execution of the commands into their own separate method